.github/workflows/development_infrastructure.yml

```diff
@@ -1158,9 +1158,6 @@ jobs:
                 APIM_TFS_URL=${{ secrets.APIM_TFS_URL }} \
                 APIM_TFS_KEY=${{ secrets.APIM_TFS_KEY }} \
                 APIM_TFS_VALUE=${{ secrets.APIM_TFS_VALUE }} \
-                MULESOFT_API_PARTY_DB_KEY="${{ secrets.MULESOFT_API_PARTY_DB_KEY}}" \
```

Comment:
I think these were missed in https://github.com/UK-Export-Finance/dtfs2/pull/1845